seeq/addons/correlation/__main__.py

```diff
@@ -198,11 +199,15 @@ def cli_interface():
         sys.exit(0)
     user = args.username
     logging_attempts(user)
-    seeq_url = args.seeq_url
+    seeq_url = get_seeq_url()
+
     if seeq_url is None:
-        seeq_url = input(f"\n Seeq base URL [{spy.client.host.split('/api')[0]}]: ")
-        if seeq_url == '':
-            seeq_url = spy.client.host.split('/api')[0]
+        seeq_url = input(f"\n Please Input Seeq base URL (eg: https://example.seeq.site): ")
+
+        os.mkdir(".urlfile")
```

Comment:
What happens if it's a reinstall and the dir already exits? this will fail

Reply:
You can probably check if the dir exists first, if not create it, then you can safely try to dump the pickle to that dir (with the line below)